### ISSUE TITLE
Uncaught ReferenceError: provide is not defined

### ISSUE THREAD

**Issue by rizkimuliono** [ajnauleau/rel-path-finder#2]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.42.0 x64
**Electron**: 4.2.7
**OS**: Mac OS X 10.15.3
**Thrown From**: [rel-path-finder](https://github.com/ajnauleau/rel-path-finder) package 1.2.0

### Stack Trace

Uncaught ReferenceError: provide is not defined

```
At /Users/rizkimuliono/.atom/packages/rel-path-finder/lib/rel-path-finder.js:48

ReferenceError: provide is not defined
    at Object.find (/packages/rel-path-finder/lib/rel-path-finder.js:48:15)
    at HTMLElement.relPathFinderFind (/packages/rel-path-finder/lib/rel-path-finder.js:24:44)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:349385)
    at CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:347860)
    at AtomEnvironment.dispatchApplicationMenuCommand (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:1:719963)
    at EventEmitter.t (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:1:727886)
    at EventEmitter.emit (events.js:182:13)
```

### Commands

```
 10x -4:09.5.0 core:backspace (input.hidden-input)
 12x -2:08.3.0 link:open (input.hidden-input)
  2x -1:54.6.0 open-path:open (input.hidden-input)
     -1:51.8.0 application:open (input.hidden-input)
  5x -1:13.6.0 core:backspace (input.hidden-input)
     -1:11.5.0 core:confirm (input.hidden-input)
  3x -0:32.2.0 core:backspace (input.hidden-input)
     -0:27.2.0 autocomplete-plus:activate (input.hidden-input)
     -0:24.9.0 core:backspace (input.hidden-input)
     -0:15.8.0 rel-path-finder:toggle (span.syntax--punctuation.syntax--definition.syntax--string.syntax--end.syntax--php)
     -0:12.9.0 core:save (input.hidden-input)
     -0:07.3.0 rel-path-finder:find (input.hidden-input)
```

### Non-Core Packages

```
atom-bootstrap3 1.2.12 
atom-bracket-highlight 0.3.2 
atom-codeigniter3-modular-snippets 0.2.2 
auto-indent 0.5.0 
autoclose-html 0.23.0 
autocomplete-paths 2.12.2 
code-peek 1.4.21 
codeigniter 0.4.0 
color-picker 2.3.0 
copy-with-style 1.1.0 
copy-with-syntax 0.0.3 
drunken-php 0.2.4 
fonts 3.9.0 
highlight-colors 0.9.0 
highlight-selected 0.16.0 
html-tag-auto-complete 0.1.1 
reindent 1.5.0 
reindent-line 0.2.0 
rel-path-finder 1.2.0 
teletype 0.13.4 
```

**Comment by rayonnunes** [comment]
Same Problem in Ubuntu 18.04.3 LTS, Atom: 1.40.1 x64
the shortcut Ctrl+Alt+F doesn't work, when i click Packages >> rel-path-finder >> Find 
the package works but show this error message

> ~/.atom/packages/rel-path-finder/lib/rel-path-finder.js:48

```
ReferenceError: provide is not defined
    at Object.find (~/.atom/packages/rel-path-finder/lib/rel-path-finder.js:48:15)
    at HTMLElement.relPathFinderFind (~/.atom/packages/rel-path-finder/lib/rel-path-finder.js:24:44)
    at CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/static/<embedded>:11:349123)
    at CommandRegistry.dispatch (/usr/share/atom/resources/app/static/<embedded>:11:347598)
    at AtomEnvironment.dispatchApplicationMenuCommand (/usr/share/atom/resources/app/static/<embedded>:1:718717)
    at EventEmitter.t (/usr/share/atom/resources/app/static/<embedded>:1:726640)
    at EventEmitter.emit (events.js:182:13)
```

**Comment by ajnauleau** [comment]
Fixed in latest version v1.3.4, things should be working after you update :)
